nanoid/method.py

```diff
@@ -2,17 +2,22 @@
 from __future__ import division
 
 from math import ceil, log
+from typing import Callable, Sequence
+from typing_extensions import TypeAlias
```

Comment:
If you add `typing_extensions` at runtime, you should also add it to the project's dependencies!
Type-checkers special-case `typing_extensions`, but it's not part of the stdlib !

You can avoid using it at runtime by doing:
```suggestion
from __future__ import annotations # move this to the top of the file

from typing import Callable, Sequence, TYPE_CHECKING
if TYPE_CHECKING:
	from typing_extensions import TypeAlias
```

or
```py
from typing import Callable, Sequence, TYPE_CHECKING
if TYPE_CHECKING:
	from typing_extensions import TypeAlias

	Algorithm: TypeAlias = Callable[[int], Sequence[int]]
else:
	Algorithm = Callable[[int], Sequence[int]]
```